### PR TITLE
Update the relevant notebook with a notation to use the latest MCT

### PR DESCRIPTION
**Pull request by Idan-BenAmi**
## Pull Request Description:
Update the relevant notebook with a notation to use the latest MCT version.

## Checklist before requesting a review:
- [x] I set the appropriate labels on the pull request.
- [x] I have added/updated the release note draft (if necessary).
- [x] I have updated the documentation to reflect my changes (if necessary).
- [x] All function and files are well documented. 
- [x] All function and classes have type hints.
- [x] There is a licenses in all file.
- [x] The function and variable names are informative. 
- [x] I have checked for code duplications.
- [x] I have added new unittest (if necessary).

### CODE REVIEW

File: tutorials/notebooks/example_keras_effdet_lite0.ipynb [hunk 1]
```diff
@@ -81,7 +80,8 @@
   {
    "cell_type": "markdown",
    "source": [
-    "In order to convert the PyTorch model, you'll need to use the conversion code in the [MCT tutorials folder](https://github.com/sony/model_optimization/tree/main/tutorials), so we'll clone the MCT repository to a local folder and only use that code. The installed MCT package will be used for quantization. "
+    "In order to convert the PyTorch model, you'll need to use the conversion code in the [MCT tutorials folder](https://github.com/sony/model_optimization/tree/main/tutorials), so we'll clone the MCT repository to a local folder and only use that code. The installed MCT package will be used for quantization. \n",
```
Comment (elad-c) [inline tutorials/notebooks/example_keras_effdet_lite0.ipynb:83]: not sure "latest version" is correct. maybe just say explicitly that you're running on the latest version of the source code

File: tutorials/notebooks/example_keras_effdet_lite0.ipynb [hunk 2]
```diff
@@ -107,6 +107,7 @@
    "source": [
     "import sys\n",
     "sys.path.insert(0,\"/content/local_mct\")\n",
+    "!pip install -r /content/local_mct/requirements.txt\n",
```
Comment (elad-c) [inline tutorials/notebooks/example_keras_effdet_lite0.ipynb:110]: maybe also run `pip uninstall mctq` and then get the latest code of mctq quantizers?

**Review by elad-c — COMMENTED** [review]
Same comments apply to all files

File: tutorials/notebooks/example_keras_effdet_lite0.ipynb (hunk 2 above)
Comment (Idan-BenAmi) [reply tutorials/notebooks/example_keras_effdet_lite0.ipynb:110]: we assume mctq version is updated in "requirements.txt"

**Review by elad-c — APPROVED** [review]

**Pull request merged**